_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
either ways, I think the answer is no.. its going to be either `assets/css/main.css` or `assets/main.css` if we're to follow Parker's Game Plan for Jekyll 3.3